udata/commands/tests/test_fixtures.py

```diff
@@ -0,0 +1,15 @@
+import json
+from tempfile import NamedTemporaryFile
+
+from udata import models
+from udata.commands.fixtures import generate_fixtures
+from udata.tests import DBTestMixin, TestCase
+
+
+class FixturesTest(DBTestMixin):
+    def test_generate_fixtures(self, cli):
+        cli("generate-fixtures")
```

Comment:
Is there a reason why you prefered to use cli instead of importing the function?

Reply:
The `generate_fixtures` being a flask-cli, it has some prerequisites:
`Error: Could not locate a Flask application. You did not provide the "FLASK_APP" environment variable, and a "wsgi.py" or "app.py" module was not found in the current directory.`

I saw that the jobs and datasets commands were tested this way, would you rather we do it differently?